packages/constants/src/log.ts

```diff
@@ -1 +1,3 @@
 export const REDACTED_TEXT = '--redacted--';
+
+export const getRedactedKeyText = (key: string) => `--redacted:${key}--`;
```

Comment:
@paescuj Is this function okay to be in the `constants` package? 👀

Reply:
Move to `utils` 👍